Jenkinsfile

```diff
@@ -85,6 +85,7 @@ pipeline {
                     steps {
                         checkPatch user: GITHUB_USER_USR,
                                    password: GITHUB_USER_PSW,
+                                   branch: "debug-patch-line-mapping",
```

Comment:
While this is very useful for working out the pylint error fixing, we wouldn't want to actually land this, so it would have to be removed before this patch can land.  And then based on how many pylint errors are left, we'd have to decide whether we want to land the _debug-patch-line-mapping_ branch of the _code_revew_ project.

We might also want to set up some temporary "ignore" settings for pylint until we close on how we want to resolve the remaining (mostly _too-many_ type issues I think) issues, so that we can re-enable pylint checking for all patches.  Or maybe we wait until we resolve that discussion and fixes before re-enabling the checking on lines that are not modified in the patch.

More topics for group discussion I guess.

Reply:
Agreed on both counts. The pylint check directive in Jenkinsfile has been removed in latest patch here.

I very much agree we want to play with our pylint settings both short and long term. Both will need some kind of group agreement. Unfortunately his PR has become quite messy to follow due to the comment spam, I was hoping some of the discussion for short term ignores could happen here.